h_transport_materials/property_database/steel_316L/steel_316L.py

```diff
@@ -144,7 +153,7 @@
 )
 
 forcey_solubility = Solubility(
-    S_0=1.50 * htm.ureg.mol * htm.ureg.m**-3 * htm.ureg.Pa**-0.5,
+    S_0=1.50 * SOLUBILITY_UNIT_MOL_VOLUME,
```

Comment:
`SOLUBILITY_UNIT_MOL_VOLUME` isn't specific enough. I understand this refactors the code a little bit, but for readability, I'd rather keep the full units expression here.
```suggestion
    S_0=1.50 * htm.ureg.mol * htm.ureg.m**-3 * htm.ureg.Pa**-0.5,
```